pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1461,4 +1461,15 @@ self: super: {
     };
   };
 
+  # Remove when https://github.com/danfran/hcoord/pull/8 is merged and a new
+  # release is published to Hackage.
+  hcoord = overrideSrc super.hcoord ({
```

Comment:
Please don't override the source for the entire package. Instead, please apply the necessary changes as patches to the official release tarball. This approach has the advantage that the patches will *fail* when a new version comes out and this is a good reminder for us to remove the override.